components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/utils/CommonUtil.java

```diff
@@ -328,4 +332,59 @@ public static String extractArchive(File sourceFile, String destination) throws
             throw new APIImportExportException(errorMessage, e);
         }
     }
+
+    /**
+     * This method will be used to generate Endpoint certificates and meta information related to endpoint certs
+     *
+     * @param filePath String of new file path
+     * @param content  String of content to write into the file
+     * @throws IOException If an error occurs when generating new certs and yaml file
+     */
+    public static void generateFiles(String filePath, String content) throws IOException {
+        FileOutputStream fos = null;
```

Comment:
Shall we use try resource when creating the FileOutputStream? So that we do not need to close the file explicitly in line 365.